### PR TITLE
Update import doco password format

### PR DESCRIPTION
**Pull request by mooreds**
Fix for https://github.com/FusionAuth/fusionauth-issues/issues/1238 

* updated the relevant field on the import api
* added a troubleshooting section for user import issues
* linked to it from the migration guide

### CODE REVIEW

File: site/docs/v1/tech/apis/_import-users-request-body.adoc [hunk 1]
```diff
@@ -100,6 +100,10 @@ The User's mobile phone number. This is useful is you will be sending push notif
 
 [field]#users``[x]``.password# [type]#[String]# [required]#Required#::
 The User's password. If [field]#encryptionScheme# is provided this value is the existing password hash, if [field]#encryptionScheme# is not provided this value is assumed to be plain text and it will be hashed during import.
++
+This field must not be a string in a Crypt/Shadow password format such as `$5a$foobarbaz$728348sehalkoiew90u32970`.
```
Comment (robotdan) [inline site/docs/v1/tech/apis/_import-users-request-body.adoc:104]: I may say something like:

> When importing a bcrypt hash, you may have a value such as `$5a$foobarbaz$728348sehalkoiew90u32970`. This single string represents the bcrypt version, factor, salt and hash.

File: site/docs/v1/tech/troubleshooting.adoc [hunk 2]
```diff
@@ -157,6 +158,16 @@ include::docs/v1/tech/account-management/_account-management-troubleshooting.ado
 
 include::docs/v1/tech/shared/_troubleshooting-uninstall-upgrade-reinstall-rpm.adoc[]
 
+== Troubleshooting User Import
+
+If you are importing password hashes, ensure you provide only the hash in the `password` import field. For some formats you'll need to separate out the hash and the salt on the link:/docs/v1/tech/apis/users/#import-users[Import API]. If you do not, the import will succeed, but the user will not be able to login.
+
+For example, a bcrypt value of `$2a$10$N9qo8uLOickgx2ZMRZoMyeIjZAgcfl7p92ldGxad68LJZdL17lhWy` would be split out to the following import fields:
```
Comment (robotdan) [inline site/docs/v1/tech/troubleshooting.adoc:165]: We could add a link to some bcrypt doc that describes why we are splitting in this manner. This wiki page has a break down of the components of this string. https://en.wikipedia.org/wiki/Bcrypt

File: site/docs/v1/tech/apis/_import-users-request-body.adoc (hunk 1 above)
Comment (mooreds) [reply site/docs/v1/tech/apis/_import-users-request-body.adoc]: Makes sense.

File: site/docs/v1/tech/troubleshooting.adoc (hunk 2 above)
Comment (mooreds) [reply site/docs/v1/tech/troubleshooting.adoc:165]: Awesome, thanks, done.

**Comment by mooreds** [comment]
Planning to merge this today, @robotdan

**Pull request merged**